src/v2/content_digest.rs

```diff
@@ -1,36 +1,47 @@
-use crate::errors::Result;
 /// Implements types and methods for content verification
 use sha2::{self, Digest};
 
 /// ContentDigest stores a digest and its DigestAlgorithm
 #[derive(Clone, Debug, PartialEq)]
-pub(crate) struct ContentDigest {
+pub struct ContentDigest {
```

Comment:
Does this need to be `pub`?

Reply:
I changed visibility to `pub` because I included it into `ContestDigestError::Verify` variant details.
I *think* this can be useful for library user.
However, I see alternative option: we can include `format!`-ed digests into error instead. this way we can keep ContestDigest private.